templates/boot_loader_conf/grub_submenu.template

```diff
@@ -1,3 +1,3 @@
-submenu '$menu_name' --class gnu-linux --class gnu --class os {
+submenu '$menu_name (Choose this for more options)' --class gnu-linux --class gnu --class os {
```

Comment:
Hmmm, looks too long for an indicator of submenu.

Reply:
now sound more like an order or pushing to do something than the note

Reply:
What about:
```suggestion
submenu '$menu_name [go to submenu]' --class gnu-linux --class gnu --class os {
```

Reply:
Since I don't have a strong opinion on that I would go for @meaksh 's solution if @vzhestkov does not object...

Reply:
Another suggestion from the internal chat by @vzhestkov 
```suggestion
submenu '$menu_name -> [submenu]' --class gnu-linux --class gnu --class os {
```